content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -316,7 +341,7 @@ where the output is similar to this:
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  creationTimestamp: 2016-02-18T18:54:22Z
+  creationTimestamp: 2022-02-18T18:54:22Z
```

Comment:
Maybe I'm missing something here, but are we bumping the dates based on some specific requirement and according to a specific pattern?

Reply:
Not really, I just like our docs to look maintained.

Reply:
Ah okay, thanks for clarifying!